src/workloads/execution/ExpressiveQueries.yml

```diff
@@ -0,0 +1,207 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+
+# Phases:
+# 1. Insert data
+# 2. Fsync
+# 3. Plain query (without index or projection)
+# 4. Query with projection, but without index
+# 5. Create index
+# 6. Fsync
+# 7. Query with projection and with index
+# 8. Drop collection
+
+- Name: InsertData
+  Type: Loader
+  Threads: 1 # Loader doesn't support multiple threads for a single collection
+  Phases:
+  - Repeat: 1
+    Database: &DB test
+    Threads: 1
+    CollectionCount: 1 # Collection name will be Collection0, this is not configurable.
+    DocumentCount: 1000000 # 1M
+    BatchSize: 1000
+    Document:
+      region: {^RandomInt: {min: 0, max: 1999}}
+      firstname: {^RandomString: {length: {^RandomInt: {min: 3, max: 10}}}}
```

Comment:
Cool -- love the nesting :) RandomString also supports an alphabet if you want to constrain it. Two other things may be useful for you:
1. ^join. This contatenates multiple strings. If you care about compressibility you can join a random string with a string of a repeated character. 
2. ^Choose has a string variant and can be nested with the join. 
3. There's a thing called ^FastRandomString -- it's semantically the same as ^RandomString, but it is faster at the cost of lower quality of randomness (not all letters will be equally likely). The quality difference often does not matter.

Reply:
Thanks for the suggestions!
1. Used it for making valid email addresses
2. I was thinking about using a pre-defined list of names, but it seems that random strings would suffice. The workload does not depend on the name values
3. Replaced all `RandomString` with `FastRandomString`